code/modules/projectiles/magazines/flamer.dm

```diff
@@ -17,9 +17,9 @@
 	var/flamer_chem = "utnapthal"
 	flags_magazine = AMMUNITION_HIDE_AMMO
 
-	var/max_intensity = 40
+	var/max_intensity = 50
 	var/max_range = 5
-	var/max_duration = 30
+	var/max_duration = 50
```

Comment:
I don't think these values need to change. They contribute to the differences between all subtypes of `/obj/item/ammo_magazine/flamer_tank` and as such will affect likely more than you want to affect.

Reply:
Still unresolved. Welcome to make a tank subtype that has more lenient limits but it shouldn't be super easy to acquire.

Reply:
fair enough, I'll make something when the research rework comes out. Can it get merged as is with just the fuels?